boards/seeed/wio_wm1110_dev_board/wio_wm1110_dev_board.dts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2025 Ishraq Ibne Ashraf <ishraq.i.ashraf@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include <nordic/nrf52840_partition.dtsi>
+#include "wio_wm1110_dev_board-pinctrl.dtsi"
+
+/ {
+	model = "Wio-WM1110 Dev Board";
+	compatible = "seeed,wio-wm1110-dev-board";
+
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		sw0 = &button0;
+		sw1 = &button1;
+		magn0 = &lis3dh;
+		accel0 = &lis3dh;
+		ambient-temp0 = &sht4x;
+	};
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led0: led_0 {
+			label = "Green LED";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+		};
+
+		led1: led_1 {
+			label = "Red LED";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+
+		button0: button_0 {
+			label = "Button 1";
+			gpios = <&gpio0 23 GPIO_PULL_UP>;
+		};
+
+		button1: button_1 {
+			label = "Button 2";
+			gpios = <&gpio0 25 GPIO_PULL_UP>;
+		};
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+
+	i2c_power {
+		gpio-hog;
+		output-high;
+		gpios = <7 GPIO_ACTIVE_HIGH>;
+	};
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uarte";
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_default>;
+	pinctrl-names = "default";
+};
+
+&i2c0 {
+	compatible = "nordic,nrf-twim";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_STANDARD>;
+	pinctrl-0 = <&i2c0_default>;
+	pinctrl-names = "default";
+
```

Comment:
this seems to be a lora board, any reason the dts file lacks any mention of lora hardware?